Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -676,6 +676,7 @@ void MediaPlayerPrivateGStreamerMSE::updateStates()
             break;
         case GST_STATE_PAUSED:
         case GST_STATE_PLAYING:
+            maybeFinishSeek();
```

Comment:
Sometimes it is not needed, because the check is performed in context of asyncStateChangeDone()